sellotape/sellotape_dj/main_app/templates/sellotape/search.html

```diff
@@ -0,0 +1,35 @@
+{% extends "user.html" %}

```

Comment:
Isn't this should be on the home page? 🤔

Reply:
This is the HTML that represents the results page, the form of the search bar is located in the navigation HTML file, as it should be located there while using the app.